### PR TITLE
Update customer-key-set-up.md

### PR DESCRIPTION
**Pull request by msbemba**
Added the requirement of an invoiced account.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2296

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Review by unwrapsodding — APPROVED** [review]

**Comment by KCCross** [comment]
Requested additional reviewers through email. Good catch.

**Comment by brdesai** [comment]
@msbemba The invoice requirement is for Azure Subscriptions that are used to create Azure Key Vaults for the Customer Key setup. If the Azure Subscription is of Pay As You Go type (especially with Credit Card) then it can not be tagged for Do Not Cancel. We need to separately call out requirements for Azure Subscriptions and the M365 licenses.

There's a separate PR that's pending publishing which @KCCross  is working on. That needs to be updated first before making this additional change for the Subscription requirement.

I am curious on what specific error the customer faced with the non-invoiced account. Can you please share those error details. Thank you.

**Comment by msbemba** [comment]
@brdesai Thank you for the response. Below is the response from the customer when they contacted Microsoft about this

Response from Customer- 

Ok, so you don't get a direct error. It isn't like you try to click something or execute one of the PS commands and it tells you 'no'.

During the early stages when you email the two teams (SPO and EXO) and provide your tenant information, you will be informed that you need an invoiced account to do the Customer Key(CK) process.

What sucks is that since this information isn't provided upfront, in the beginning of the documentation for CK, it causes a long delay as you go about becoming invoiced.

The processes for invoiced are to do it at time of purchase, change your billing method in the admin consoles, OR if invoice is not available to you in the console, you have to do a new purchase, and then start a tenant to tenant migration. Except that isn't fully true, if you start a support ticket and get someone that isn't just going to tell you to do a new purchase, you will get forwarded to the billing team, and they will work with the credit team to do the credit check required for invoiced billing. This can be a long process (9 weeks for us), and company (or personal) information will be shared, and tenant information will be changed to fit wording and information expected to be seen by the credit team.
Now, there are two credit teams. There is one for 365 and one for Azure, so this must be done with both teams, and their process is not synced.

Once you get to invoiced, either through a new purchase or through a drawn-out process getting to the credit teams, you can then continue the path of setting up CK with the SPO and EXO teams.



---

Thanks
Sri

**Comment by brdesai** [comment]
@msbemba Thanks for the info.

File: microsoft-365/compliance/customer-key-set-up.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ You must set up Azure before you can use Customer Key for Office 365. This topic
   
 ## Before you set up Customer Key
 
-Before you get started, ensure that you have the appropriate licensing for your organization. Customer Key in Microsoft 365 is offered in Office 365 E5 or the Advanced Compliance SKU. To understand the concepts and procedures in this topic, review the [Azure Key Vault](https://docs.microsoft.com/azure/key-vault/) documentation. Also, become familiar with the terms used in Azure, for example, [tenant](https://docs.microsoft.com/previous-versions/azure/azure-services/jj573650(v=azure.100)).
+Before you get started, ensure that you have the appropriate licensing for your organization and there is an invoiced account setup. Customer Key in Microsoft 365 is offered in Office 365 E5 or the Advanced Compliance SKU. To understand the concepts and procedures in this topic, review the [Azure Key Vault](https://docs.microsoft.com/azure/key-vault/) documentation. Also, become familiar with the terms used in Azure, for example, [tenant](https://docs.microsoft.com/previous-versions/azure/azure-services/jj573650(v=azure.100)).
```
Comment (KCCross) [inline microsoft-365/compliance/customer-key-set-up.md]: and that your account is invoiced and not paid for with a credit card.

Comment (msbemba) [reply microsoft-365/compliance/customer-key-set-up.md]: Made the changes as per the request.

**Comment by yogkumgit** [comment]
@KCCross Please reply. Thanks

**Pull request merged**